terraso_backend/apps/graphql/schema/__init__.py

```diff
@@ -172,6 +177,15 @@ class Mutations(graphene.ObjectType):
     mark_project_seen = ProjectMarkSeenMutation.Field()
     update_soil_data = SoilDataUpdateMutation.Field()
     update_depth_dependent_soil_data = DepthDependentSoilDataUpdateMutation.Field()
+    update_soil_data_depth_interval = SoilDataUpdateDepthIntervalMutation.Field()
+    delete_soil_data_depth_interval = SoilDataDeleteDepthIntervalMutation.Field()
+    update_project_soil_settings = ProjectSoilSettingsUpdateMutation.Field()
+    update_project_soil_settings_depth_interval = (
+        ProjectSoilSettingsUpdateDepthIntervalMutation.Field()
+    )
+    delete_project_soil_settings_depth_interval = (
+        ProjectSoilSettingsDeleteDepthIntervalMutation.Field()
+    )
```

Comment:
suggestion: order these to match L21-25.

Reply:
L21-25 are ordered alphabetically, the rest of the mutations are ordered by mutation type, with the ordering being `add update archive delete` and this is consistent with that